src/modules/commons/components/checkbox/Checkbox.js

```diff
@@ -63,6 +63,10 @@ export class Checkbox extends BaElement {
 			this._onToggle(event);
 		};
 
+
+		// const root = document.querySelector('ba-checkbox').shadowRoot;
+		// root.addEventListener('click', alert('click'));
```

Comment:
this commented out code needs an additional comment, otherwise it should be removed

Reply:
thx, done